src/operations/indexes.ts

```diff
@@ -189,18 +187,18 @@ export class CreateIndexesOperation extends CommandOperation<CreateIndexesOption
 
     const cmd: Document = { createIndexes: this.collectionName, indexes };
 
-    if (options.commitQuorum != null) {
+    if (this.commitQuorum != null) {
       if (serverWireVersion < 9) {
         callback(
           new MongoError('`commitQuorum` option for `createIndexes` not supported on servers < 4.4')
         );
         return;
       }
-      cmd.commitQuorum = options.commitQuorum;
+      cmd.commitQuorum = this.commitQuorum;
     }
 
     // collation is set on each index, it should not be defined at the root
-    this.options.collation = undefined;
+    // this.collation = undefined; FIXME!!!!!!
```

Comment:
What's going on here?

Reply:
The `Successfully create index with collation` test is failing in CI, probably related to this?

Reply:
Yep fixed this, this was actually some cruft from TS conversion worth mentioning to spread the knowledge about it (@reggi), we used this syntax in few places:
```ts
class A {
  prop!: string;
}
```
because at the time the errors surrounding prop being optional were numerous and it felt right to assert the prop's presence, I just pushed changing props on `CommandOperation` that were asserted to be optional and there are no more type errors surrounding that. So we removed the dependence on the properties with further work that followed the big operations layer conversion.